marklogic-data-hub/src/main/resources/ml-modules/root/data-hub/5/flow/job-query-lib.sjs

```diff
@@ -59,6 +59,43 @@ function findStepResponses(query) {
   return response;
 }
 
+function valuesExist(values) {
+  return !(values || values.length);
+}
+
+function findJobDocuments({start, pageLength, jobId, status, startTimeBegin, startTimeEnd, endTimeBegin, endTimeEnd, user}) {
```

Comment:
Seems fine so far, but just keep in mind whether /v1/search would be simpler. Search options have the benefit of being self-documenting too - i.e. our docs can say "Just look at host:8013/v1/config/query/hubJobDocuments" instead of having to spell out all the parameters and also maintain that list.